cli/plasmo/src/features/manifest-factory/base.ts

```diff
@@ -245,28 +245,36 @@ export abstract class BaseFactory<T extends ExtensionManifest = any> {
     return this
   }
 
-  toggleContentScript = async (path?: string, enable = false) => {
+  isPathInvalid = (path?: string): path is undefined => {
```

Comment:
affirmativeive `isPathValid` and the upstream should decide if they wanna add a `!`. Otherwise, code is full of double negatives

Reply:
What do you mean?

Reply:
should be `isPathValid` (functions that have an `is` at the beginning should be affirmative never `isNot`) then if we wanna use this in the future to figure out if it is valid, we would have to do `!isPathInvalid` which is two negatives (harder to reason about)

Reply:
Discussed on call - the positive convention doesn't make sense here since this fx is mainly used within this class to check, the negative negative cases is "uncommon" for this call,